default.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 var DefaultCaller = func() interface{} {
-	return fmt.Sprintf("%+v", stack.Caller(4))
+	return fmt.Sprintf("%+v", stack.Caller(3))
```

Comment:
I don't understand this change right now - what am I missing that makes this change necessary?

Reply:
The parameter indicates the number of stack frames below the current frame to display. This function is called in `micrologger` now instead of in `go-kit`, so we don't have to go as far down the stack to get the caller.

Reply:
Okay, makes total sense :+1: